src/rules/enforce-assert-throws.ts

```
@@ -1,25 +1,119 @@
+          case AST_NODE_TYPES.VariableDeclaration:
+            node.declarations.forEach((decl) => walk(decl));
+            break;
+          case AST_NODE_TYPES.VariableDeclarator:
+            if (node.init) {
+              walk(node.init);
+            }
+            break;
+          default:
+            // Handle other node types generically
+            for (const key of Object.keys(node)) {
+              if (key === 'parent' || key === 'range' || key === 'loc') continue;
```

ESLint

[error] Insert `⏎···············`

(prettier/prettier)